testlib/infra.py

```diff
@@ -89,8 +89,22 @@ def check_result(result, format_str, format_str_args):
                     f"{os.path.join(VAR_TMP, mountpoint_dir, name)}: {err}"
                 )
 
+    # Unset MergeScheduled
+    for (fs_path, name, origin), pool_name in StratisDbus.fs_list().items():
```

Comment:
Here, I think you must as well destructure more, like:

```
for (fs_path, name, (origin_set, _)), pool_name in StratisDbus.fs_list().items():
  if origin_set:
    ...
```

Reply:
OK, should be resolved.